src/Errata/Internal/Render.hs

```diff
@@ -21,10 +23,10 @@ module Errata.Internal.Render
     , renderSourceLines
     ) where
 
-import qualified GHC.Arr as A
+import           Control.Arrow
+import           Control.Applicative
 import           Data.List
-import qualified Data.List.NonEmpty as N
-import qualified Data.Map.Strict as M
+import qualified Data.IntMap as I
```

Comment:
Put this in alphabetical order please! One day I'll have a formatter I like and enforce it, but not today.